common/src/test/java/com/microsoft/identity/common/internal/providers/microsoft/nativeauth/utils/ApiConstants.kt

```diff
@@ -0,0 +1,20 @@
+package com.microsoft.identity.common.internal.providers.microsoft.nativeauth.utils
+
+import java.net.URL
+
+interface ApiConstants {
```

Comment:
Javadoc

Reply:
Yes javadoc or comments are good even in test code so that reviewers and future readers of the code can understand the test code

Reply:
Done